docs/TOC.yml

```diff
@@ -897,6 +901,8 @@
               href: macios/system-special-folders.md
             - name: Universal links
               href: macios/universal-links.md
+            - name: Xcode Sync
```

Comment:
```suggestion
            - name: Xcode sync
```

Reply:
Headings/titles should always use title case. The entry above is in the Mac Cat section. It also needs adding to the iOS section (just above Mac Cat).